src/extension.ts

```diff
@@ -498,8 +515,9 @@ export function activate(context: vscode.ExtensionContext) {
           name: collectionName,
           scopeName: node.scopeName,
         });
-
+        
         clusterConnectionTreeProvider.refresh(node);
+        clusterConnectionTreeProvider.refresh(node.parentNode);
```

Comment:
I think you could replace the tree provider refresh call above instead of doing a second call.

Reply:
I test that too but doesn't work that way so need to refresh both.

Reply:
The tree provider should be executing in flow-down style. So I would expect refreshing the parent to update all child nodes too. What did you see?

Reply:
@MikeGoldsmith Same error is seeing here too.
![Screenshot 2023-02-20 at 12 59 00 PM](https://user-images.githubusercontent.com/42893909/220044501-0eb25bfc-465e-401f-a4ca-76101df9c181.png)

Reply:
The looks like it cannot find a local file for a cluster document. Not sure how if that's related or not.

Either way, we can accept as it is and come back as an optimisation if required.